command/sub8_missions/nodes/tp.yaml

```diff
@@ -0,0 +1,6 @@
+bin: -24 14 -3.5
```

Comment:
Yeah since it was in that original warning that I removed I didn't think I needed to put it there. I'll do so now.